app/Listeners/Smartcars/EvaluateFlightCriteria.php

```diff
@@ -57,7 +58,26 @@ public function handle(BidCompleted $event)
             return;
         }
 
+        $pirepTime = $this->minutes($pirep->flight_time);
+
+        $networkTime = NetworkData::where('disconnected_at', '>', $posreps->first()->created_at)
+            ->where('connected_at', '<', $posreps->last()->created_at)
+            ->sum('minutes_online');
+
+        if ((($networkTime / $pirepTime) * 100 ) < 90) {
+            $pirep->markFailed('You were not connected to the VATSIM network.', null);
```

Comment:
Should follow the same format as the other failure reasons, prefixed with 'Failed:'